tests/examples/aws-java-webserver/app/build.gradle

```diff
@@ -25,7 +25,7 @@ repositories {
     }
 }
 
-var pulumiJavaSdkVersion = System.getenv("PULUMI_JAVA_SDK_VERSION") ?: "0.0.1"
+var pulumiJavaSdkVersion = System.getenv("PULUMI_JAVA_SDK_VERSION") ?: "0.2.1"
```

Comment:
This actually may break something intentional. We use 0.0.1 to mark the "Current version in repo" so that `make install_sdk` works and we can reference that from examples as the default; like with this change `make test_example.xyz` will go and test against the released 0.2.1 version not your changes.

I wonder how to organize this better.

Reply:
hmm, that's unfortunate because it breaks testing locally. IMO current version should be always generated from pulumicli (hash + dirty in case of active debug, maybe even some kind of build number that will change on very compile) so things are consistent and does not require intimate knowledge of our build system

Reply:
I'll remove this change then. I thought it was an omission.